src/Resources/app/administration/src/module/sw-in-app-purchases/component/sw-in-app-purchase-checkout/index.ts

```diff
@@ -34,12 +37,6 @@ export default Shopware.Component.wrapComponentConfig({
     },
 
     computed: {
-        priceModel(): IAP.InAppPurchasePriceModel | null {
-            return this.inAppPurchaseCart?.positions?.[0].feature.priceModel || null;
-        },
-        purchase(): IAP.InAppPurchase | null {
-            return this.inAppPurchaseCart?.positions?.[0].feature || null;
-        },
```

Comment:
this is a BC, the component is not private. Please mark as such (fine with the BC)

Reply:
It's in the extension store. None of these components are marked as private/internal. @cyl3x stated that because of components are part of Extension store BC's are fine, since we don't want people to overwrite/extend this code anyway.

Reply:
I was wrong. I see none of the IAP stuff is private. But other stuff is. I'll add it